source/jormungandr/tests/ptref_tests.py

```diff
@@ -221,6 +245,8 @@ def test_route(self):
         assert com[0]['type'] == 'standard'
         assert com[0]['value'] == "I'm a happy comment"
 
+        self._test_links(response, 'routes')
```

Comment:
would have been nice to check on the second page of an API to see if previous is OK.